app/controllers/events_controller.rb

```diff
@@ -8,7 +8,7 @@ class EventsController < ApplicationController
     render json: {error: 'Google Server error'}, status: :service_unavailable
   end
 
-  rescue_from Google::Apis::ClientError, GoogleEvent::InvalidParamsError do |exception|
+  rescue_from Google::Apis::ClientError, GoogleCalendar::Adding::InvalidParamsError do |exception|
```

Comment:
I would change the name of the exception to `GoogleCalendar::AddEventInvalidParamsError`